operations/network/gateway_testing.md

```diff
@@ -40,11 +50,58 @@ test-networks:
 ```
 
 For each network, the test will attempt to obtain a token from Keycloak. On an NCN, it should be able to get a token
-from each of those networks. It will then use that token to attempt to access each of the services defined in
-`gateway-test-defn.yaml`, on each of the `test-networks`. The test will determine whether it should or should not be
-able to access the service, and it will output a `PASS` or `FAIL` for each service, as appropriate.
+from NMNLB, CMN, and either CAN or CHN depending on how the system is configured. It will then use that token to attempt to
+access each of the services defined in `gateway-test-defn.yaml`, on each of the `test-networks`. The test will determine whether
+it should or should not be able to access the service, and it will output a `PASS` or `FAIL` for each service, as appropriate.
+At the end of the tests it will compile and output a final overall PASS/FAIL status.
+
+## Running gateway tests on a UAN or a Compute Node
+
+The same set of tests will be run from a UAN or Compute Node by executing the following command from an NCN that has the `docs-csm` RPM installed.  The hostname of the UAN or Compute Node under tests must be specified.
+
+### UAN Test execution
+
+```bash
+ncn# /usr/share/doc/csm/scripts/operations/gateway-test/uan-gateway-test.sh <uan-hostname>
+```
+
+### Compute Node Test execution
+
+```bash
+ncn# /usr/share/doc/csm/scripts/operations/gateway-test/cn-gateway-test.sh <cn-hostname>
+```
+
+Both scripts will fetch the admin client secret, the configured user network, and the site domain from the system.
```

Comment:
This paragraph seems to be talking about both UAN and Compute. If that's the case, probably a header should be added before it, since right now it looks like it is part of the Compute Node section.

Reply:
I moved the lower paragraphs above the two different command execution headings.